tools/src/commands/GenerateBareApp.ts

```diff
@@ -291,6 +302,33 @@ async function createScripts({ projectDir }) {
   console.log('Added package scripts!');
 }
 
+async function stageAndCommitInitialChanges({ projectDir }) {
+  await spawnAsync('git', ['init'], { cwd: projectDir });
+  await spawnAsync('git', ['add', '.'], { cwd: projectDir });
+  await spawnAsync('git', ['commit', '-m', 'Initialized bare app!'], { cwd: projectDir });
```

Comment:
We already have some nice tools to operate on Git repositories, see https://github.com/expo/expo/blob/main/tools/src/Git.ts